src/foam/nanos/cron/SimpleIntervalSchedule.js

```diff
@@ -4,18 +4,29 @@
  * http://www.apache.org/licenses/LICENSE-2.0
  */
 
- foam.CLASS({
+foam.CLASS({
   package: 'foam.nanos.cron',
   name: 'SimpleIntervalSchedule',
 
   implements: [
     'foam.nanos.cron.Schedule'
   ],
 
+  messages: [
+    { name: 'START_DATE_ERROR', message: 'Start Date must be after today' },
+    { name: 'ENDS_ON_ERROR', message: 'End Date must be after start date' },
+    { name: 'INVALID_DATE_ERROR', message: 'Please provide the date' }
+  ],
+
   properties: [
     {
       class: 'Date',
-      name: 'startDate'
+      name: 'startDate',
+      validateObj: function(startDate) {
+        if ( ! startDate ) return this.INVALID_DATE_ERROR;
+        // check against current date
+        if ( startDate <= new Date() ) return this.START_DATE_ERROR;
```

Comment:
Are we not allowing a schedule to start today?

Reply:
no, we are allowing a start date after today.

Reply:
So what we could do is allow them to select today, but getNextScheduledDate can't return today. Wouldnt that be still sus though vs just enforcing the schedule to start tmrw. Remember @nanoNeel's comments about making a schedule which starts today. What I guess could happen is they could select today and the schedule can try to execute today but somewhere on the transaction side blocks it from going in. But the schedule itself doesn't know. @nanoNeel @sarthak-marwaha ?

Reply:
Oh yeah, well I guess we can force it to start tomorrow, don't think it is that big of an issue since we are only allowing date setting not time. We just allow nextScheduledDate to also be tomorrow if that is valid